README.md

```diff
@@ -0,0 +1,55 @@
+PDF Generator: A Plugin for Pelican
+====================================================
+
+[![Build Status](https://img.shields.io/github/workflow/status/pelican-plugins/pdf/build)](https://github.com/pelican-plugins/pdf/actions)
+[![PyPI Version](https://img.shields.io/pypi/v/pelican-pdf)](https://pypi.org/project/pelican-pdf/)
+![License](https://img.shields.io/pypi/l/pelican-pdf?color=blue)
+
+The PDF Generator plugin automatically exports articles and pages as PDF files as part of the site generation process.
+PDFs are saved to: `output/pdf/`
+
+Installation
+------------
+
+This plugin can be installed via:
+
+    python -m pip install pelican-pdf
+
+Usage
+-----
+
+To customize the PDF output, you can use the following settings in your
+Pelican configuration file::
```

Comment:
We can probably remove (1) the hard line wrap and (2) the extra colon on the end of the sentence.

Reply:
Indeed, Copy & Paste issue from the previous RST based Readme